pandas/core/resample.py

```diff
@@ -13,7 +13,7 @@
 from pandas.tseries.frequencies import to_offset, is_subperiod, is_superperiod
 from pandas.core.indexes.datetimes import DatetimeIndex, date_range
 from pandas.core.indexes.timedeltas import TimedeltaIndex
-from pandas.tseries.offsets import DateOffset, Tick, Day, _delta_to_nanoseconds
+from pandas.tseries.offsets import DateOffset, Tick, Day, delta_to_nanoseconds
```

Comment:
pandas.tseries.offsets is a public-ish module. In general I would not add non-underscored internal methods to that namespace like `delta_to_nanoseconds`

Reply:
Yah, if nothing else `delta_to_nanoseconds` should be imported directly from tslibs.timedelta.  I'll put something together.

Reply:
On second thought, do we need to obscure the name if it isn't in `__all__`?

Reply:
Well, tab completion (= for me my interaction with the API) does not take into account `__all__` I think.